mapApp/static/mapApp/js/index.js

```diff
@@ -159,15 +187,16 @@ var collisionsUnfiltered = collisions,
 newInfrastructuresUnfiltered = newInfrastructures;
 $("input.slider").on("slideStop", function (e) { filterPoints(e.value[0], e.value[1]) });
```

Comment:
While outside the scope of this PR, I noticed that when I drag on the date filter slider, the map gets dragged as well. I wonder if there is an input.slider.on("slide" or "drag") event we can capture and stop propagation to the map?

Reply:
good catch, this was also happening when the mouse was over the legend which also seemed like a bug to me. Added a solution for both in with the section that disables zooming when over the legend (fd5ff1)